disnake/reaction.py

```diff
@@ -64,7 +64,8 @@ def __init__(
         emoji: Optional[Union[PartialEmoji, Emoji, str]] = None,
     ) -> None:
         self.message: Message = message
-        self.emoji: Union[PartialEmoji, Emoji, str] = emoji or message._state.get_reaction_emoji(
+        # _get_emoji_from_data won't return None
+        self.emoji: Union[PartialEmoji, Emoji, str] = emoji or message._state._get_emoji_from_data(  # type: ignore
```

Comment:
but why do we need a type ignore?

Reply:
reaction emojis always have at least an ID or name, so this always returns *something*